Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 1269, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/util.py", line 60, in just_run
    return loop.run_until_complete(coro)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/asyncio/base_events.py", line 654, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 707, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 1025, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.11.9/x64/lib/python3.11/site-packages/nbclient/client.py", line 919, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from buildingmotif.ingresses.bacnet import BACnetNetwork

bacnet = BACnetNetwork("172.24.0.1/32") # don't change this if you are using the docker compose setup
for rec in bacnet.records:
    print(rec)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[3], line 1[0m
[0;32m----> 1[0m [38;5;28;01mfrom[39;00m [38;5;21;01mbuildingmotif[39;00m[38;5;21;01m.[39;00m[38;5;21;01mingresses[39;00m[38;5;21;01m.[39;00m[38;5;21;01mbacnet[39;00m [38;5;28;01mimport[39;00m BACnetNetwork
[1;32m      3[0m bacnet [38;5;241m=[39m BACnetNetwork([38;5;124m"[39m[38;5;124m172.24.0.1/32[39m[38;5;124m"[39m) [38;5;66;03m# don't change this if you are using the docker compose setup[39;00m
[1;32m      4[0m [38;5;28;01mfor[39;00m rec [38;5;129;01min[39;00m bacnet[38;5;241m.[39mrecords:

File [0;32m~/work/BuildingMOTIF/BuildingMOTIF/buildingmotif/ingresses/__init__.py:6[0m
[1;32m      4[0m [38;5;28;01mfrom[39;00m [38;5;21;01mbuildingmotif[39;00m[38;5;21;01m.[39;00m[38;5;21;01mingresses[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcsvingress[39;00m [38;5;28;01mimport[39;00m CSVIngress  [38;5;66;03m# noqa[39;00m
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mbuildingmotif[39;00m[38;5;21;01m.[39;00m[38;5;21;01mingresses[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnaming_convention[39;00m [38;5;28;01mimport[39;00m NamingConventionIngress  [38;5;66;03m# noqa[39;00m
[0;32m----> 6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mbuildingmotif[39;00m[38;5;21;01m.[39;00m[38;5;21;01mingresses[39;00m[38;5;21;01m.[39;00m[38;5;21;01msemantic_graph_synthesis[39;00m [38;5;28;01mimport[39;00m (  [38;5;66;03m# noqa[39;00m
[1;32m      7[0m     SemanticGraphSynthesizerIngress,
[1;32m      8[0m )
[1;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01mbuildingmotif[39;00m[38;5;21;01m.[39;00m[38;5;21;01mingresses[39;00m[38;5;21;01m.[39;00m[38;5;21;01mtemplate[39;00m [38;5;28;01mimport[39;00m (  [38;5;66;03m# noqa[39;00m
[1;32m     10[0m     TemplateIngress,
[1;32m     11[0m     TemplateIngressWithChooser,
[1;32m     12[0m )
[1;32m     14[0m [38;5;28;01mtry[39;00m:

[0;31mModuleNotFoundError[0m: No module named 'buildingmotif.ingresses.semantic_graph_synthesis'
ModuleNotFoundError: No module named 'buildingmotif.ingresses.semantic_graph_synthesis'

